lib/eclipse/tests/data/integration_tests/SCHEDULE/SCHEDULE_WELLS2

```diff
@@ -130,5 +130,25 @@ DATES              -- 12
 /      
 
 WCONINJE 
-     'W_1'     'WATER'  'SHUT'      'RATE'  20000.000  * 123 678 4* /
+     'W_1'     'WATER'  'OPEN'      'RATE'  20000.000  * 123 678 4* /
+/
+
+DATES              -- 13
```

Comment:
The indices `12`, `13` ... should probably be put after the `DATES` and before the `WCON*` keywords.  For that case, there should be no `15`, which agrees well with what the test shows.